### PR TITLE
Template for .env added

### LINKED ISSUE #26: Create example.env file
Document the .env variables by creating a .env.example file.

Can you please assign this issue to me for gssoc 2023?

### PR DESCRIPTION
**Pull request by SINJINI-G**
fixes #26

- added the .env.example file to be used as a template for the original .env file (updated)

Can you please review the PR and add the gssoc tag in #26, #24? 
@Adi-ty @Divyansh3021 

Thank you!

### CODE REVIEW

File: server/.env.example [hunk 1]
```diff
@@ -0,0 +1,9 @@
+DATABASE_HOST="YOUR MYSQL DATABASE HOST"
+DATABASE_PORT="YOUR MYSQL DATABASE PORT"
+DATABASE_USER="YOUR MYSQL DATABASE USER"
+DATABASE_PASSWORD="YOUR MYSQL DATABASE PASSWORD"
+DATABASE_NAME="YOUR MYSQL DATABASE NAME"
+EMAIL_SENDER="SENDER'S EMAIL"
+EMAIL_PASSWORD="SENDER'S EMAIL PASSWORD"
+EMAIL_ME2="RECIEVER EMAIL"
```
Comment (Adi-ty) [inline server/.env.example:8]: Instead of "RECIEVER EMAIL" make it "SAMPLE EMAIL1" and "SAMPLE EMAIL2"

**Review by Adi-ty — CHANGES_REQUESTED** [review]

File: server/.env.example (hunk 1 above)
Comment (SINJINI-G) [reply server/.env.example]: Updated the required changes.. Kindly review @Adi-ty

**Comment by Adi-ty** [comment]
Great thanks @SINJINI-G

**Comment by SINJINI-G** [comment]
Thank you for accepting the PR.. will you please add the gssoc tag?

**Pull request merged**